source/pug/partials/events.pug

```diff
@@ -4,7 +4,9 @@
     h4 Launch events
 
 .container
-  include:markdown ../../markdown/events.md
+  .row.justify-content-center
+    .col-12.col-xl-10
```

Comment:
`.row` and `.col-*` aren't being used consistently on this repo. So I think we're okay without it. If we do wanna use it here let's use `col-xl-12` instead since on mockup this section takes 100% width.